### PR TITLE
Admin Login feature added

### PR DESCRIPTION
**Pull request by Lingtan1112**
##### Admin Login Feature added

### CODE REVIEW

File: src/main/java/in/lingtan/service/UserService.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import in.lingtan.validator.UserValidator;
+
+public class UserService {
+	
+	private UserService() {
+		//Default constructor
+	}
+
+	static Map<String, String> adminCredentialHashTable = new HashMap<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/lingtan/service/UserService.java]: private final

File: src/main/java/in/lingtan/servlet/AdminValidationServlet.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import in.lingtan.service.UserService;
+
+/**
+ * Servlet implementation class AdminValidationService
+ */
+@WebServlet("/AdminValidationServlet")
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/lingtan/servlet/AdminValidationServlet.java]: Rename for AdminLoginServlet or LoginServlet

File: src/main/java/in/lingtan/validator/UserValidator.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+		//Default constructor
+	}
+	
+	/**
+	 * This method validates whether the employeeId is in valid Format or not i.e) valid id : Ling2657
+	 * @param employeeId
+	 * @return
+	 */
+	
+	public static boolean isValidEmployyeeIdFormat(String employeeId) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/lingtan/validator/UserValidator.java]: Check spelling

**Review by ghost — CHANGES_REQUESTED** [review]
Added comments

File: src/main/webapp/index.jsp [hunk 4]
```diff
@@ -1,20 +1,16 @@
 <%@page import="java.util.Date"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp:1]: Remove unused imports

File: src/test/java/in/lingtan/validator/ValidatorTestCase.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+		assertTrue(isValidPasswordLength);
+	}
+	
+	/**
+	 * Valid employeeId format is validated
+	 */
+	@Test
+		public void isValidFormatEmployyeeIdValidTest() { 
+		String employeeName = "Lin2657";
+			boolean isValidEmployeeId = UserValidator.isValidEmployyeeIdFormat(employeeName);
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/lingtan/validator/ValidatorTestCase.java:77]: Format the code

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**